packages/shared/src/utils/validation/declarative-validation-rules.js

```diff
@@ -19,7 +19,7 @@ export const validLength = (value = '', options) =>
     (options.min ? value.length >= options.min : true) && (options.max ? value.length <= options.max : true);
 export const validPassword = value => /(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]+/.test(value);
 const validBarrier = value => /^[+-]?\d+\.?\d*$/.test(value);
-const validEmail = value => /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,63}$/.test(value);
+export const validEmail = value => /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,63}$/.test(value);
```

Comment:
Let's move it to line 21 to other exported variables above

Reply:
![image](https://user-images.githubusercontent.com/82078941/160543056-b497972c-b857-418e-a6df-19113cd045a7.png)